src/main/java/dan200/computercraft/shared/peripheral/diskdrive/DiskDrivePeripheral.java

```diff
@@ -96,6 +136,11 @@ public final boolean hasAudio()
         return media != null && media.getAudio( stack ) != null;
     }
 
+    /**
+     * Returns the title of the inserted audio disk.
+     *
+     * @return The title of the audio, or {@code nil} if no audio disk is inserted.
+     */
     @LuaFunction
     public final Object getAudioTitle()
```

Comment:
Again, a `@Nullable` here. Sorry, these should really have been here originally.